docs/getting-started/tutorial.md

```diff
@@ -67,9 +67,10 @@ The `main.jsx` file is the entry point. Open it up and we'll put React Router on
 
 👉 **Add [`DataBrowserRouter`][databrowserrouter] and a [`Route`][route] to `main.jsx`**
 
-```jsx lines=[3,8-10] filename=src/main.jsx
+```jsx lines=[4, 9-11] bad lines=[3] filename=src/main.jsx
```

Comment:
not sure if this is how you highlight that you are moving a line 😅  I couldn't test it

Reply:
Hi @timdorr 

Sorry, I wanted to test this syntax highlighting and it is not doing what I thought 😅 
![Screenshot 2022-08-23 at 09 57 42](https://user-images.githubusercontent.com/17012976/186117296-5d21ac8f-2ceb-4ff7-a500-cdc7ef1d358e.png)

I am raising another PR to fix it unless you know how to highlight just line 3 to be removed